en/developer_guide/logs.rst

```diff
@@ -0,0 +1,97 @@
+Logs
+====
+
+Our backoffice logging is based on `Monolog`_.
+Open Orchestra provides an interface to display all the user actions on the website.
+
+.. image:: ../../images/log_preview.png
+
+Configuration
+-------------
+
+Monolog uses channels to record the log. To link a channel and a record, you will have to define an handler.
+The configuration of the handler can be placed in the ``app/config/log.yml`` file.
+
+For instance, if you want to store the logs in mongo, you will have to define a handler with the database connection information.
+The level entry indicates the information type to record in the logs.
+Each level is a number which value is standardized in the PSR-3 recommendation.
+For example 200 it's for info. The different level are defined in the `Monolog documentaion`_
+
+.. code-block:: yaml
+
+        mongo:
+            type: mongo
+            level: 200
+            mongo:
+                host: "localhost"
+                database: "%open_orchestra_cms.mongodb.database%"
+                collection: log
+            channels: [openorchestra]
+
+
+The channels key defines the channels where are written the logs.
+If no channel is indicated then the logs are written in all channels.
+It's possible to put multiple channels or exclude one.
+The services who have the channel ``openorchestra`` write only in this chanel
+and our handler shows logs who are in the ``openorchestra`` channel.
+
+The different possibility of configuration about channels are explained in the `channels documentation`_.
+
+Processor
+---------
+
+The loggers allowing to record logs know the date and time but not the IP address and user name.
+To save IP address and user name, Open Orchestra has a processor.
```

Comment:
I would more say : 

``When you publish your log, some information may be missing. Monolog allows you to add a processor which will fill up the logging information by adding some context.
To do it, you can create a service, tagged by monolog.processor. You can referer to the Monolog processor documentation for further exemple.``